### PR TITLE
send dex alerts to team bigmac

### PR DESCRIPTION
**Pull request by anvddriesch**
Towards: https://github.com/giantswarm/giantswarm/issues/26583

Rainbow on-call ends by the end of this week so let's route the dex related alerts to bigmac from now on.

### Checklist

- [ ] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

**Review by vvondruska — APPROVED** [review]

**Comment by gawertm** [comment]
@anvddriesch is that a manual overwrite because we didn't move the owership of components yet (codeowners) or are these alerting rules anyways not respecting the codeowners?

**Comment by gawertm** [comment]
I haven't done the PR yet and not merged. Lucas is on onsite this week and I wanted to wait for his feedback

**Comment by anvddriesch** [comment]
That makes sense!

**Pull request merged**
